src/encoded/static/components/search.js

```diff
@@ -765,48 +765,51 @@ class DateSelectorFacet extends React.Component {
     }
 
     componentDidMount() {
-        this.setActiveFacetParameters();
+        this.setActiveFacetParameters(true);
     }
 
-    setActiveFacetParameters() {
+    setActiveFacetParameters(initializationFlag) {
+        let activeFacet = null;
+        let activeFilter = null;
+        // If there is a date filter applied, we'll use that filter to set state when the component is mounted
+        if (initializationFlag) {
+            // if a date range has already been selected, we will use that date range to populate drop-downs
+            const existingFilter = this.props.filters.filter(filter => filter.field === 'advancedQuery');
```

Comment:
I think this would be a more elegant solution but I was looking at the MDN page for `.find` and noticed that there's no IE support for it. So I think I'll keep `.filter` for now.